.travis.yml

```diff
@@ -37,6 +21,23 @@ cache:
         - $HOME/.gradle/wrapper/
         - $HOME/.android/build-cache
 
+before_install:
+  - mkdir "$ANDROID_HOME/licenses" || true
+  - echo -e "\n8933bad161af4178b1185d1a37fbf41ea5269c55\nd56f5187479451eabf01fb78af6dfcb131a6481e" > "$ANDROID_HOME/licenses/android-sdk-license"
+  - echo -e "\n84831b9409646a918e30573bab4c9c91346d8abd\n504667f4c0de7af1a06de9f4b1727b84351f2910" > "$ANDROID_HOME/licenses/android-sdk-preview-license"
+
+android:
+  components:
+    # https://github.com/travis-ci/travis-ci/issues/6040#issuecomment-219367943
+    - tools
+    - tools
+
+before_script: >
+  echo y | ${ANDROID_HOME}tools/bin/sdkmanager --channel=3
```

Comment:
nit: can you add a comment explaining this param? maybe also link to the docs https://developer.android.com/studio/command-line/sdkmanager.html

Reply:
SGTM